packages/esm-appointments-app/src/patient-appointments/patient-appointments-header.scss

```
@@ -0,0 +1,37 @@
+@use '@carbon/styles/scss/type';
+@use '@carbon/colors';
+@use '@carbon/styles/scss/spacing';
```

Nit, you can use the `layout` namespace instead here:

```suggestion
@use '@carbon/layout';
```

It would require changing all occurrences of the `spacing.spacing-xx` namespace to `layout.spacing-xx` instead.